frame/democracy/src/benchmarking.rs

```diff
@@ -206,11 +207,12 @@ benchmarks! {
 		let origin = T::CancellationOrigin::successful_origin();
 		let referendum_index = add_referendum::<T>(0)?;
 		let call = Call::<T>::emergency_cancel(referendum_index);
-		assert!(Democracy::<T>::referendum_status(referendum_index).is_ok());
+		assert_ok!(Democracy::<T>::referendum_status(referendum_index));
 	}: { call.dispatch_bypass_filter(origin)? }
 	verify {
 		// Referendum has been canceled
-		assert!(Democracy::<T>::referendum_status(referendum_index).is_err());
+		assert_noop!(Democracy::<T>::referendum_status(referendum_index),
+            Error::<T>::ReferendumInvalid);
```

Comment:
```suggestion
		assert_noop!(
			Democracy::<T>::referendum_status(referendum_index),
			Error::<T>::ReferendumInvalid,
		);
```

Reply:
Probably other such formatting things, wont look at them all.

Reply:
I thought this is the coding style, I can go over the changes and fix it everywhere.